src/lib.rs

```diff
@@ -33,6 +85,17 @@ where
     T: AsPrimitive<usize> + FromPrimitive + Ord + PrimInt + ToPrimitive + Unsigned,
     usize: AsPrimitive<T>,
 {
+    /// Constructs a new SparseVec from a vector (with a user-defined backing storage type) where
```

Comment:
I think this should be "Constructs a new `SparseVec` from a `Vec` of unsigned integers." [The "user-defined backing storage" bit isn't relevant here.]